gossip-base/src/main/java/org/apache/gossip/GossipSettings.java

```diff
@@ -45,6 +45,9 @@
   
   private String activeGossipClass = "org.apache.gossip.manager.SimpleActiveGossipper";
   
+  private String transportManagerClass = "org.apache.gossip.transport.UdpTransportManager";
```

Comment:
`UdpTransportManager` will not be available once it's in a different module. 

Ideally, Gossip will not work unless a transport manager is specified, but I was going for non-breaking at this point.